### ISSUE TITLE
ValueError: Unable to configure handler 'logging.handlers.NTEventLogHandler'

### ISSUE THREAD

**Issue by Deadshot20** [totalgood/nlpia#23]
Hey,

I have been reading chapter 4 and while I am trying to work around the example mentioned in section 4.3 (about SVD), I am getting this error:

`from nlpia.book.examples.ch04_catdog_lsa_sorted import lsa_models, prettify_tdm`

`ValueError: Unable to configure handler 'logging.handlers.NTEventLogHandler': __init__() missing 1 required positional argument: 'appname'`

Here is the full error trace:

```
TypeError                                 Traceback (most recent call last)
C:\ProgramData\Anaconda3\lib\logging\config.py in configure(self)
    557                     try:
--> 558                         handler = self.configure_handler(handlers[name])
    559                         handler.name = name

C:\ProgramData\Anaconda3\lib\logging\config.py in configure_handler(self, config)
    730         try:
--> 731             result = factory(**kwargs)
    732         except TypeError as te:

TypeError: __init__() missing 1 required positional argument: 'appname'

During handling of the above exception, another exception occurred:

ValueError                                Traceback (most recent call last)
<ipython-input-1-a1787124f80a> in <module>()
----> 1 from nlpia.book.examples.ch04_catdog_lsa_sorted import lsa_models, prettify_tdm

~\AppData\Roaming\Python\Python36\site-packages\nlpia\book\examples\ch04_catdog_lsa_sorted.py in <module>()
    105 from sklearn.feature_extraction.text import TfidfVectorizer
    106 
--> 107 from nlpia.data.loaders import get_data
    108 
    109 pd.options.display.width = 120

~\AppData\Roaming\Python\Python36\site-packages\nlpia\data\__init__.py in <module>()
----> 1 from nlpia.loaders import *  # noqa

~\AppData\Roaming\Python\Python36\site-packages\nlpia\loaders.py in <module>()
     61 from pugnlp.util import clean_columns
     62 
---> 63 from nlpia.constants import DATA_PATH, BIGDATA_PATH
     64 from nlpia.constants import DATA_INFO_FILE, BIGDATA_INFO_FILE, BIGDATA_INFO_LATEST
     65 from nlpia.constants import INT_MIN, INT_NAN, MIN_DATA_FILE_SIZE

~\AppData\Roaming\Python\Python36\site-packages\nlpia\constants.py in <module>()
     96 
     97 
---> 98 logging.config.dictConfig(LOGGING_CONFIG)
     99 logger = logging.getLogger(__name__)
    100 

C:\ProgramData\Anaconda3\lib\logging\config.py in dictConfig(config)
    793 def dictConfig(config):
    794     """Configure logging using a dictionary."""
--> 795     dictConfigClass(config).configure()
    796 
    797 

C:\ProgramData\Anaconda3\lib\logging\config.py in configure(self)
    564                         else:
    565                             raise ValueError('Unable to configure handler '
--> 566                                              '%r: %s' % (name, e))
    567 
    568                 # Now do any that were deferred

ValueError: Unable to configure handler 'logging.handlers.NTEventLogHandler': __init__() missing 1 required positional argument: 'appname'
```

I have no clue what's going wrong here, please help.

**Comment by 1922353531** [comment]
I have the same problem.

**Comment by hobson** [comment]
Fixed with release 0.2.8. Need to double check on a Windows machine before closing this ticket.
